docs/first-order/classify_waveforms.ipynb

```diff
@@ -0,0 +1,2327 @@
+{
```

Comment:
<p>If you want users to do this with FS and RS, write another line <code>units_spike_times = unit_spike_times_rs</code> and comment it out. then add another comment telling the user they can swap these lines. also mention it in the markdown. you can remove the  <code>#units_spike_times = units['spike_times']</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AllenInstitute/openscope_databook/pull/281/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>